pkg/plugin/response.go

```diff
@@ -84,6 +85,16 @@ func parseStreamResponse(reader io.Reader) backend.DataResponse {
 			return newResponseError(err, backend.StatusInternal)
 		}
 		labelsField.Append(d)
+
+		if timeFd.Len() == 0 {
+			lineField.Append(string(d))
+		}
+	}
+
+	if timeFd.Len() == 0 {
+		for i := 0; i < lineField.Len(); i++ {
+			timeFd.Append(time.Now())
```

Comment:
timestamp should be calculated once. Otherwise, lines could have different timestamps.

Another question: why this check happens here and not at L:89?

Reply:
The frame should have the same number of elements for each field. If one of the fields has a different number of values, the plugin shows the error because we should correctly show the lines per each request.
For example, if timeFd has ten values and labelsField has nine values, we will have an error saying that the number of values can't differ for each frame. The frame is representing one record in the UI

Reply:
> For example, if timeFd has ten values and labelsField has nine values, we will have an error saying that the number of values can't differ for each frame

Do we have a test case for this?